bugbug/model.py

```diff
@@ -193,6 +212,13 @@ def train(self, importance_cutoff=0.15):
 
         print(f"X: {X.shape}, y: {y.shape}")
 
+        if isinstance(y[0], np.ndarray):
+            is_multilabel = True
+
+        # Don't sort class_names for multilabel models as we'll lose information for predictions
```

Comment:
Maybe we can solve this by sorting in the bugtype model itself (by sorting KEYWORD_LIST)

Reply:
Can you check?

Reply:
works as expected!